Runner.php

```diff
@@ -20,24 +20,47 @@ public function __construct($config, $server)
 
     public function queueDirectory($dir, $prefix = '')
     {
+        if(isset($this->config['include_paths'])) {
+            $this->queueWithIncludePaths();
+        } else {
+            $this->queuePaths($dir, $prefix, $this->config['exclude_paths']);
+        }
+
+        $this->server->process_work(false);
+    }
+
+    public function queueWithIncludePaths() {
+        foreach ($this->config['include_paths'] as $f) {
+            if ($f !== '.' and $f !== '..') {
+
+                if (is_dir("/code$f")) {
```

Comment:
I have the same question/concern here as on https://github.com/codeclimate/codeclimate-phpcodesniffer/pull/16/files#diff-65f87d47425b92cdeea437cd228536fdR31